### ISSUE TITLE
TRUST4 can be use for some non-tumor diseases?

### ISSUE THREAD

**Issue by YMSWhuang** [liulab-dfci/TRUST4#140]
Hello,
Thank you for this very useful tools. I have a question, this method TRUST4 can be use for some inflammatory disease, such as vasculitis,by their blood samples. Thanks.
Best
Hongbiao

**Comment by mourisl** [comment]
Yes, you can use TRUST4 for non-tumor samples. We have applied TRUST4 to healthy PBMC data for evaluations and got meaningful results. Please let me know if you found issues when running TRUST4.

**Comment by YMSWhuang** [comment]
Thanks, I will run TRUST4 for my PBMC data recently. If I found any problems which I can't deal with, maybe I will ask again :)

**Comment by YMSWhuang** [comment]
Hello,
Thanks for your rapid reply last times. Now I have another question. I want to download the data from GEO website. But the format of data isn't fastq or BAM, the address is "https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE63881", So can I use TRUST4 to analysis this data? 

Supplementary file | Size | Download | File type/resource
-- | -- | -- | --
GSE63881_RAW.tar | 26.2 Mb | (http)(custom) | TAR
GSE63881_non-normalized.txt.gz | 96.6 Mb | (ftp)(http) | TXT
Raw data are available on Series record
Processed data included within Sample table

Supplementary file	Size	Download	File type/resource
GSE63881_RAW.tar	26.2 Mb	(http)(custom)	TAR
GSE63881_non-normalized.txt.gz	96.6 Mb	(ftp)(http)	TXT
Raw data are available on Series record
Processed data included within Sample table

 Custom GSE63881_RAW.tar archive:
Supplementary file	File size
GPL10558_HumanHT-12_V4_0_R1_15002873_B.txt.gz	13.1 Mb
GPL10558_HumanHT-12_V4_0_R2_15002873_B.txt.gz	13.1 M

**Comment by mourisl** [comment]
GEO files usually only contain gene expression matrix and have no sequence (nucleotides) information. TRUST4 needs the raw sequence information either in the raw fastq file or aligned BAM file to find CDR3s.

**Comment by YMSWhuang** [comment]
Thanks for your reply. Maybe I need to find another resources.
